doc/man1/openssl-s_client.pod.in

```diff
@@ -303,14 +303,21 @@ see L<openssl-passphrase-options(1)>.
 
 The verify depth to use. This specifies the maximum length of the
 server certificate chain and turns on server certificate verification.
-Currently the verify operation continues after errors so all the problems
+Unless the B<-verify_return_error> option is given,
+the verify operation continues after errors so all the problems
 with a certificate chain can be seen. As a side effect the connection
 will never fail due to a server certificate verify failure.
 
+By default, validation of server certificates and their chain
+is done w.r.t. the (D)TLS Server (C<sslserver>) purpose.
+For details see L<openssl-verification-options(1)/Certificate Extensions>.
+
+
```

Comment:
Just one blank line between paragraphs.

Reply:
Thank you for the review and approval.
Should I better add a fixup commit (removing the extra newline) here, which would automatically invalidate the approvals,
or, e.g., as part of #18764 (after rebasing that PR on this one when it is merged)?

Reply:
Do a fixup here, my approval still stands.  It's such a trivial change, I'd assume @t8m's would also stand.